Source/MDFastBinding/Private/BindingDestinations/MDFastBindingDestinationBase.cpp

```diff
@@ -2,11 +2,17 @@
 
 #include "MDFastBindingInstance.h"
 #include "BindingValues/MDFastBindingValueBase.h"
+#include "Utils/MDFastBindingTraceHelpers.h"
 
 void UMDFastBindingDestinationBase::InitializeDestination(UObject* SourceObject)
 {
+#if defined(MDFASTBINDING_CONDENSED_PROFILING) && MDFASTBINDING_CONDENSED_PROFILING
```

Comment:
Instead of checking for `MDFASTBINDING_CONDENSED_PROFILING` all over, could you change all the instances to always use `MD_TRACE_CPUPROFILER_EVENT_SCOPE_FUNCTION_TEXT` and then #define it based on `MDFASTBINDING_CONDENSED_PROFILING` in `MDFastBindingTraceHelpers.h`?

I think it ends up being cleaner that way.